src/domain/individualApplication/IndividualApplicationPage.tsx

```diff
@@ -23,6 +23,9 @@ import OfferCard, { OfferCardProps } from './offerCard/OfferCard';
 export enum CUSTOMER_GROUP {
   PRIVATE = 'PRIVATE',
   COMPANY = 'COMPANY',
+  INTERNAL = 'INTERNAL',
+  NON_BILLABLE = 'NON_BILLABLE',
+  OTHER = 'OTHER',
```

Comment:
should this enum member be renamed too?

Reply:
my point is that `OTHER` makes sense as `OrganizationType`, since it is "other type of organization". but when it comes to customer groups, here it might make things clearer if we explicitly state this this is `other organization` and not `other customer group` 🙂